extensions/blocks/tiled-gallery/view.scss

```diff
@@ -20,7 +22,8 @@ $tiled-gallery-max-column-count: 20;
 			@for $cols from 1 through $tiled-gallery-max-column-count {
```

Comment:
If we wanna add gutter support for Circle and Square layouts with this CSS-class method, we'd have to generate 50*20 1000 times this CSS. :-(